test/command/suite/sharding/logical_range_filter/columns/stage/filtered/window_function/month/time_classify_mix.expected

```diff
@@ -80,5 +80,27 @@ logical_range_filter Logs   --shard_key timestamp   --columns[day].stage filtere
 #|d| [logical_range_filter][select] <Logs_201703>: dynamic columns are used
 #|d| [logical_range_filter][window] <Logs_201703>: [2017/03/15 00:00:00,2017/03/16 00:00:00)
 #|d| [logical_range_filter][window] <Logs_201703>: [2017/03/16 00:00:00,2017/03/17 00:00:00)
+#>logical_range_filter --columns[day].flags "COLUMN_SCALAR" --columns[day].stage "filtered" --columns[day].type "Time" --columns[day].value "time_classify_day(timestamp)" --columns[five_minute].flags "COLUMN_SCALAR" --columns[five_minute].stage "filtered" --columns[five_minute].type "Time" --columns[five_minute].value "time_classify_minute(timestamp, 5)" --columns[n_records_per_day].flags "COLUMN_SCALAR" --columns[n_records_per_day].stage "filtered" --columns[n_records_per_day].type "UInt32" --columns[n_records_per_day].value "window_count()" --columns[n_records_per_day].window.group_keys "day" --columns[n_records_per_five_minute].flags "COLUMN_SCALAR" --columns[n_records_per_five_minute].stage "filtered" --columns[n_records_per_five_minute].type "UInt32" --columns[n_records_per_five_minute].value "window_count()" --columns[n_records_per_five_minute].window.group_keys "five_minute" --filter "price >= 300" --limit "3" --logical_table "Logs" --offset "1" --output_columns "_id,n_records_per_day,n_records_per_five_minute,price" --post_filter "n_records_per_day >= 4 || n_records_per_five_minute >= 3" --shard_key "timestamp"
+#:000000000000000 filter(11): Logs_201703.price greater_equal 300
+#:000000000000000 filter(3): between(#<accessor _key(anonymous).timestamp(Logs_201703)>, "2017/03/15 00:00:00", "include", "2017/03/16 00:00:00", "exclude")
+#:000000000000000 columns[day](3)
+#:000000000000000 columns[five_minute](3)
+#:000000000000000 columns[n_records_per_day](3)
+#:000000000000000 columns[n_records_per_five_minute](3)
+#:000000000000000 filter(0): 2147483651.n_records_per_day greater_equal 4
+#:000000000000000 filter(3): 2147483651.n_records_per_five_minute greater_equal 3
+#:000000000000000 post_filter(3): Logs_201703.n_records_per_day >= 4 || n_records_per_five_minute >= 3
```

Comment:
It this expected?

Reply:
Already coverted by `filter()`. I'll fix.